src/event/ListenerMethodTags.php

```diff
@@ -31,4 +31,5 @@ final class ListenerMethodTags{
 	public const HANDLE_CANCELLED = "handleCancelled";
 	public const NOT_HANDLER = "notHandler";
 	public const PRIORITY = "priority";
+	public const NO_CONCURRENT_CALL = "noConcurrentCall";
```

Comment:
Concurrent call is actually a less intuitive behavior. I would rather have concurrent handlers opt-in.

btw it is generally an antipattern to use negative names in any programming symbol. Prefer using `exclusiveCall` if we stick with opt-in excusive.